src/bandit-evaluator.ts

```diff
@@ -144,7 +144,12 @@ export class BanditEvaluator {
     let currTopScore: number | null = null;
     let currTopAction: string | null = null;
     actionScoreEntries.forEach(([actionKey, actionScore]) => {
-      if (currTopScore === null || actionScore > currTopScore) {
+      if (
+        currTopScore === null ||
+        currTopAction === null ||
```

Comment:
this superfluous check is to appease TypeScript

Reply:
🙇‍♂️ bow to the wishes of the TS Language Server.